app/api.py

```diff
@@ -39,21 +41,32 @@ async def version():
 
 @API.get("/collections")
 async def collections():
+    """Names of collections and the count of documents in each collection
+    <pre><code>@return JSON[str,int]</pre></code>"""
     return {"result": API.db.get_database_info()}
 
 
 @API.post("/read/mentor")
 async def read_mentor(data: Optional[Dict] = None):
+    """Displays mentor(s) who meet provided criteria. Displays all mentors if no input provided
+    <pre><code>@param data: JSON[Mentor criteria]
+    @return JSON[Mentors]</pre></code>"""
     return {"result": API.db.read("Mentors", data)}
 
 
 @API.post("/read/mentee")
 async def read_mentee(data: Optional[Dict] = None):
+    """Displays mentee(s) who meet provided criteria. Displays all mentees if no input provided
+    <pre><code>@param data: JSON[Mentee criteria]
+    @return JSON[Mentees]</pre></code>"""
```

Comment:
This should indicate the return is a list of Mentee objects:
*Note: Mentee is singular.
```JSON[List[Mentee]]```